packages/web/docs/src/pages/docs/gateway/other-features/upstream-reliability.mdx

```diff
@@ -0,0 +1,212 @@
+---
+description: Improve reliability of Hive Gateway with timeout and retry policy for upstream requests
```

Comment:
Maybe this article should also include our recommended setup?

Reply:
Humm, I mean, the setup is highly variable depending on upstream implementation and client requirements.

The recommendation is to setup retry and timeout, the actual values are up to the user.

Or you want to show of very simple example with both retry and timeout enabled ?

Reply:
I think it is a bit tricky to provide some values because it is very specific to user's configuration/setup.
For retry, `Retry-After` header is the best way to control the gateway's behavior. For timeout, mayyybe we can give a value like 5 seconds or sth but still too tricky to recommend something in  my opinion.